### PR TITLE
Edit server address

### PR DESCRIPTION
**Pull request by ovekyc**

### CODE REVIEW

**Comment by ochanje210** [comment]
LGTM

File: shared/components/CreateForm.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ import TForm from 'tcomb-form-native';
 import RNFS from 'react-native-fs';
 import {Actions} from 'react-native-router-flux';
 
-const API_SETITEMS = 'http://goober.herokuapp.com/api/items';
+const API_SETITEMS = 'http://ec2-52-78-96-168.ap-northeast-2.compute.amazonaws.com/api/items';
```
Comment (ochanje210) [inline shared/components/CreateForm.js]: components/CreateForm.js is outdated and removed. please see components/Create.js

File: shared/utils.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+export const queryBuilder = (arr) => {
+  /* arr is array of Object which form is {key:-, value:-} */
+  const strings = arr.map((obj) => {
+    return `${obj.key.toString()}=${obj.value.toString()}`;
+  });
+  const queryString = strings.reduce((first, second) => {
+    return `${first}&${second}`;
+  });
+  return `?${queryString}`;
+};
```
Comment (yuda1124) [inline shared/utils.js]: it is very good kk

**Comment by yuda1124** [comment]
LGTM

**Comment by ochanje210** [comment]
LGTM

**Pull request merged**